python/ray/dag/compiled_dag_node.py

```diff
@@ -163,26 +178,11 @@ def _exec_task(self, task: "ExecutableTask", idx: int) -> bool:
         output_val = method(*resolved_inputs)
         output_writer.write(output_val)
     except Exception as exc:
-        # TODO(rui): consider different ways of passing down the exception,
-        # e.g., wrapping with RayTaskError.
-        output_writer.write(RayDAGTaskError(exc))
+        output_writer.write(_wrap_exception(exc))
```

Comment:
Catch `RayChannelError` for writing to channel

Reply:
Ah good calls, thanks!